charts/sentry-apps/values.yaml

```
@@ -0,0 +1,78 @@
+spec:
+  metadata:
+    # spec.metadata.name -- name for the ArgoCD application
+    name: sentry
+    # spec.metadata.namespace -- namespace for the ArgoCD application
+    namespace: argocd
+  destination:
+    # spec.destination.namespace -- namespace for Sentry
+    namespace: sentry
+  # spec.project -- project to deploy the ArgoCD application to
+  project: default
+  source:
+    # spec.source.chart -- name of the Chart for Sentry
+    chart: sentry
+    # spec.source.path -- path of the Chart for Sentry when using Git repository
+    path: ""
+    # spec.source.repoURL -- Chart museum to get Sentry
+    repoURL: "https://sentry-kubernetes.github.io/charts"
+    # spec.source.targetRevision -- revision of the chart to use for Sentry
+    targetRevision: 4.7.2
+
+user:
+  # user.create -- if true, creates the user defined by email and password.
+  create: true
+  # user.email -- Admin user email
+  email: admin@sentry.local
+  # user.password -- Admin user password
+  password: admin
```

Should we mandate changing this by not defining it to ensure that it is "secure by default"?

---

To my understandings, it is expected that the first install changes it.

https://github.com/sentry-kubernetes/charts/blob/develop/sentry/values.yaml#L6

However it could be generated. A nice to have imho.

---

I can't seem to find the code that changes it. If we keep it empty in our chart we would be forced to override it when deploying through sentry:

```suggestion
  password: 
```
We could also set `user.create=false` per default so it doesn't get created w/o explicitly setting values.

We might want to default it to `user.create=false` on our prod anyway since we already have users that we can import into the db.

---

The code doesn't change the password, a human have to do so.

If the user already exists, it won't fail.

- https://github.com/sentry-kubernetes/charts/blob/5cec61417e243ce49fe509f6bc62847add0975d0/sentry/templates/hooks/user-create.yaml#L40
- https://github.com/getsentry/sentry/blob/a66938b106f85f25d9590b6371338b8499b7a6cb/src/sentry/runner/commands/createuser.py#L87